### ISSUE TITLE
[Alan] Scene 7 - Voice Control example/issue not the strongest

### ISSUE THREAD

**Issue by MicheleAWilliams-A11y** [w3c/wai-people-use-web-videos#157]
Currently the primary voice control issue is not having the correct accessible name on an element. Though I know this is a WCAG item, often this is overcome (or not even used) via "Show numbers" or "Show grid" style of use. (That is, I've known a few voice control users who don't even try the direct commands.)

A few thoughts to beef this up:
- One of the more difficult issues is when items rely on mouse hover to work such as a menu icon appearing on hover because the designers feel leaving it persistent on the screen is too much visually. Since VC users aren't moving the cursor, they don't even know these exist and also have a hard time activating them. I concede this may be harder to explain in a video like this, however, plus people may not be using this trend as much since it also doesn't work for mobile devices and mobile web is growing. 
- Icon buttons present a challenge related to our current direct command example because the name is not visible (or only visible on hover). Notably that image buttons can also make it hard to know where to put the cursor for the grid mode due to lack of border around it and inability to activate the mouse hover before clicking like mouse users. Perhaps this example could work as  middle ground?

**Comment by nitedog** [comment]
Changed this to icon buttons because hover state is not specific to voice control, it impacts all keyboard-only users.
